src/librender/scene.cpp

```diff
@@ -227,6 +301,14 @@ Scene<Float, Spectrum>::sample_emitter_direction(const Interaction3f &ref, const
     return { ds, spec };
 }
 
+MTS_VARIANT Float
+Scene<Float, Spectrum>::pdf_emitter(UInt32 /*index*/, Mask /*active*/) const {
+    size_t emitters_size = m_emitters.size();
+    if (emitters_size == 0)
+        return 0.f;
+    return ek::rcp(emitters_size);
```

Comment:
It's not clear to me that `rcp` of an integer will give the right thing (`rcp` returns the same type IIRC, i.e. an integer!). This generally isn't an Enoki array, so using `ek::rcp` also feels strange. I would just write `1.f / m_emitters.size()`